crates/aptos-faucet/src/main.rs

```diff
@@ -528,4 +502,33 @@ mod tests {
         res1.unwrap();
         res2.unwrap();
     }
+
+    async fn get_client() -> (FaucetClient, JoinHandle<()>) {
+        let (_accounts, service) = setup(None);
+        let endpoint = service.endpoint().clone();
+        let (address, future) = warp::serve(routes(service)).bind_ephemeral(([127, 0, 0, 1], 0));
```

Comment:
Should this listen address be configurable? If not, at least make it 0.0.0.0 by default?

Reply:
This is in the test module, so local should be preferred